TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -518,12 +529,6 @@ void TTSSpeaker::createPipeline() {
         return;
```

Comment:
Do we need to keep intel specific implementation now?

Reply:
Removed the intel specific code.